packages/evolution-frontend/src/components/inputs/InputCheckbox.tsx

```diff
@@ -273,11 +281,15 @@ export class InputCheckbox<CustomSurvey, CustomHousehold, CustomHome, CustomPers
             )
             : null;
 
+        const shouldDisplayAsRows =
+            this.props.widgetConfig.alignment === undefined || this.props.widgetConfig.alignment === 'auto';
```

Comment:
In this one, the undefined should be displayed as rows.

1- Is it normal or a mistake? If normal, please add a comment explaining why in the code
2- The alignment type says `alignment?: 'vertical' | 'horizontal' | 'auto'`. I think you should add a `||` for the horizontal case here.